http-kernel-fixtures/sub-folder/cookie_page4.php

```diff
@@ -1,7 +1,10 @@
 <?php
     /** @var \Symfony\Component\HttpFoundation\Request $request */
     $resp = new Symfony\Component\HttpFoundation\Response();
-    $cookiePath = dirname($request->server->get('REQUEST_URI')) . '/';
+    $requestUri = $request->server->get('REQUEST_URI');
+    assert(is_string($requestUri));
+    $cookiePath = dirname($requestUri) . '/';
+    assert(is_string($cookiePath));
```

Comment:
The `dirname` function returns a string and concatenating 2 strings would also produce a string. I think, that PhpStan is clever enough to figure that out as well without a need for a `assert(is_string($cookiePath));` code.

Reply:
Nice catch...I guess that's a blind copy&paste mistake.